Tools/adb_upload_to_bebop.sh

```diff
@@ -34,7 +34,7 @@ adb shell mount -o remount,rw /
 adb shell touch /home/root/parameters
 adb shell mkdir -p /data/ftp/internal_000/fs/microsd
 
-${RPI_TOOLCHAIN_DIR}/gcc-linaro-arm-linux-gnueabihf/bin/arm-linux-gnueabihf-strip \
+${PX4_TOOLCHAIN_DIR}/gcc-linaro-arm-linux-gnueabihf/bin/arm-linux-gnueabihf-strip \
```

Comment:
Why bother stripping before upload? Either let cmake do this (it can find arm-linux-gnueabihf-strip) or consider not bothering.